### ISSUE TITLE
🐛Right-Toolbar Buttons Are Shown Wrong After a Reload

### ISSUE THREAD

**Issue by alexanderkasten** [process-engine/bpmn-studio#1359]
#### Describe your issue

After you have reloaded the diff view and navigated to the detail view with the status bar buttons the right toolbar buttons are shown wrong (browser and electron).

#### How can others reproduce the issue?

- goto diff view
- reload the page
- click on `Show Diagram`
- look at the right toolbar

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [ ] I've checked if this issue already exists
- [ ] I've included all the information that i think is relevant
- [ ] I've added logs and/or screenshots (if applicable)
- [ ] I've mentioned PRs and issues that relate to this one

**Comment by Vyperus** [comment]
@alexanderkasten 
I was not able to reproduce this.

The buttons are always displayed correctly even though I wonder what "wrong" means in this context anyway.
Can you provide more insight on this?

**Comment by robpal94** [comment]
Also, when opening the diff view and reloading, the following error is thrown: 
```
bluebird.core.js:1408 Unhandled rejection Error: required args <xml=string>
    at error (file:///Applications/bpmn-studio.app/Contents/Resources/app.asar/scripts/vendor-bundle.js:12972:16)
    at Parser.parse (file:///Applications/bpmn-studio.app/Contents/Resources/app.asar/scripts/vendor-bundle.js:13166:23)
...
bluebird.core.js:1408 Unhandled rejection Error: required args <xml=string>
    at error (file:///Applications/bpmn-studio.app/Contents/Resources/app.asar/scripts/vendor-bundle.js:12972:16)
    at Parser.parse (file:///Applications/bpmn-studio.app/Contents/Resources/app.asar/scripts/vendor-bundle.js:13166:23)
    at file:///Applications/bpmn-studio.app/Contents/Resources/app.asar/scripts/vendor-bundle.js:14364:24
```

**Comment by Vyperus** [comment]
Update: I was able to reproduce this and I will look into this.

**Comment by Vyperus** [comment]
Will be fixed in release v4.22.0

**Comment by robpal94** [comment]
Shouldn't we keep the issue open if this still needs to be fixed?

**Comment by Vyperus** [comment]
Didn't we fix this in v4.22.0. If not please reopen this Issue.
